raving-remoras/bangazon-site
# Login Fail

# Description
Better login flow. If login is incorrect, the login form reloads with an error message.

## Bug Fixes
Adds document.ready to jQuery functions called on Add Product

## Steps to Test Solution

1. Try an incorrect login.
2. Make sure City and State are hidden on create product view.

## Testing
